modules/programs/firefox.nix

```diff
@@ -262,6 +262,111 @@ in {
         '';
       };
 
+      languagePacks = mkOption {
+        type = types.listOf (types.enum [
+          "ach"
+          "af"
+          "an"
+          "ar"
+          "ast"
+          "az"
+          "be"
+          "bg"
+          "bn"
+          "br"
+          "bs"
+          "ca-valencia"
+          "ca"
+          "cak"
+          "cs"
+          "cy"
+          "da"
+          "de"
+          "dsb"
+          "el"
+          "en-CA"
+          "en-GB"
+          "en-US"
+          "eo"
+          "es-AR"
+          "es-CL"
+          "es-ES"
+          "es-MX"
+          "et"
+          "eu"
+          "fa"
+          "ff"
+          "fi"
+          "fr"
+          "fy-NL"
+          "ga-IE"
+          "gd"
+          "gl"
+          "gn"
+          "gu-IN"
+          "he"
+          "hi-IN"
+          "hr"
+          "hsb"
+          "hu"
+          "hy-AM"
+          "ia"
+          "id"
+          "is"
+          "it"
+          "ja"
+          "ka"
+          "kab"
+          "kk"
+          "km"
+          "kn"
+          "ko"
+          "lij"
+          "lt"
+          "lv"
+          "mk"
+          "mr"
+          "ms"
+          "my"
+          "nb-NO"
+          "ne-NP"
+          "nl"
+          "nn-NO"
+          "oc"
+          "pa-IN"
+          "pl"
+          "pt-BR"
+          "pt-PT"
+          "rm"
+          "ro"
+          "ru"
+          "sco"
+          "si"
+          "sk"
+          "sl"
+          "son"
+          "sq"
+          "sr"
+          "sv-SE"
+          "szl"
+          "ta"
+          "te"
+          "th"
+          "tl"
+          "tr"
+          "trs"
+          "uk"
+          "ur"
+          "uz"
+          "vi"
+          "xh"
+          "zh-CN"
+          "zh-TW"
+        ]);
```

Comment:
`fur`, `sat`, `sc` and `tg` seem to be missing on a cursory look over what 127 supports.

Reply:
I just copied it from the `nixpkgs` repository. Maybe it's just better to leave it at `listOf str` and write in the description how to find the value?

Reply:
Given that the list of languages is not static, and making a mistake is easily noticed, I think this is a reasonable option.